ui/src/views/Tasks/TaskIndex/ListNamespaces/index.jsx

```diff
@@ -138,7 +154,14 @@ export default class ListNamespaces extends Component {
     return (
       <Dashboard
         title="Index Browser"
-        helpView={<HelpView description={description} />}>
+        helpView={<HelpView description={description} />}
+        search={
```

Comment:
@helfi92  If a user searches for a task that doesn't exist, the `IndexedTask` component will be rendered with the error message. If they want to retry (search again), they have go back to the Index page again. So, I decided to add the search bar here as well.